packages/paypal-js/types/v6/components/billing.d.ts

```diff
@@ -0,0 +1,14 @@
+import {
+    OnApproveData,
+    OnCompleteData,
+    OnCancelData,
+    OnErrorData,
+} from "./payments";
+
+export type BillingSessionInputs = {
+    onApprove?: (data: OnApproveData) => Promise<void>;
```

Comment:
I just checked and technically `orderId`, `billingToken`, and `payerId` are what get returned for this legacy billing agreement without purchase flow.
```suggestion
    onApprove?: (data: OnApproveDataOneTimePayments) => Promise<void>;
```